sdp-utils.h

```diff
@@ -228,6 +228,12 @@ janus_sdp *janus_sdp_parse(const char *sdp, char *error, size_t errlen);
  * @returns 0 in case of success, a negative integer otherwise */
 int janus_sdp_remove_payload_type(janus_sdp *sdp, int pt);
 
+/*! \brief Helper method to check from attribute status of FEC associate to  specific codec
```

Comment:
See above: please remove the new method.

Reply:
Ok